src/main/java/Calendar1/Cal1.java

```diff
@@ -0,0 +1,11 @@
+package Calendar1;
+
+public class Cal1 {
+   public static int calculate(int [] a){
+       int result = 0;
+       for(int i = 0; i < a.length; i++){
+          result += ((int) ((Math.ceil(a[i] / 3))) - 2);
```

Comment:
What is this formula?

Reply:
Why this formula is wrong? Taking numbers from array, dividing from 3, rounding to ceiling, and minus two.

Reply:
I didn't say it's wrong, just asked what it is